daml/Demo.daml

```diff
@@ -194,4 +200,38 @@ issuance = do
       assetId = optionDesc.assetId
       quantity = 1000.0
   (_issuanceCid, optionDepositCid) <- submit common.custodian $ exerciseCmd alice.issuanceServiceCid $ Issuance.CreateIssuance createReqCid
+
+  -- Alice originates a bond
+  origReqCid <- submit alice.customer $ exerciseCmd alice.issuanceServiceCid Issuance.RequestOrigination with
+      assetLabel = "NFLX 4 7/8 04/15/23"
+      description = "NFLX 4 7/8 04/15/23"
+      safekeepingAccountId = alice.safekeepingAccountId
+      observers = []
+      claims =  C.serialize $ cpb 1000.0 (4.0 + 7.0/8.0) usdId (couponDates 2021 2023 [Jan,Jul] 15)
+  cpbDescCid :: _ <- submit common.custodian $ exerciseCmd alice.issuanceServiceCid $ Issuance.Originate origReqCid
+  Some cpbDesc <- queryContractId common.custodian cpbDescCid
+
+  -- pedagogical example from https://ledgy.com/blog/examples-of-convertible-notes/
+  let convertibleNote : Id -> Decimal -> Id -> Decimal -> Date -> Decimal -> Decimal -> Id -> C.Claim O.Observation Date Id
+      convertibleNote note principal ccy discount maturity interest cap underlying = 
+        C.when (C.at maturity) (
+          C.cond (O.observe underlying.label O.<= O.pure cap) (
+             C.scale (O.pure (principal * (1.0 + interest) / (1.0 - discount)) O./ O.observe underlying.label) (C.one underlying)
+          )( C.scale (O.pure (principal * (1.0 + interest))) (C.one ccy)
+          )
+        )
+
+  -- Alice originates a convertible note
+  origReqCid <- submit alice.customer $ exerciseCmd alice.issuanceServiceCid Issuance.RequestOrigination with
+      assetLabel = "NFLX convertible 06/21/21"
+      description = "NFLX convertible 06/21/21"
+      safekeepingAccountId = alice.safekeepingAccountId
+      observers = []
+      claims = C.serialize $ convertibleNote nflxId 100_000.0 usdId 0.20 (date 2021 Jun 06) 0.04 550.0 nflxId
+  -- FIXME: the below appears to be failing for some inexplicable reason; accept the request in the UI manually!
+  -- cnDescCid :: _ <- submit common.custodian $ exerciseCmd alice.issuanceServiceCid $ Issuance.Originate origReqCid
+  -- Some cnDesc <- queryContractId common.custodian cnDescCid
```

Comment:
@GeorgSchneider I think there is a bug in the origination choice, and it's returning an empty value.

For the time being, I've commented out, so you have to click the 'accept' button in the Origination screen on the UI, so that this appears in the 'Instruments' section.